examples/common/python/utility/utility.py

```diff
@@ -206,7 +216,7 @@ def decrypted_response(input_json_str, encrypted_session_key):
         data = item['data'].encode('UTF-8')
         e_key = item['encryptedDataEncryptionKey'].encode('UTF-8')
         if not e_key or (e_key == "null".encode('UTF-8')):
-            encryption_key_byte = encrypted_session_key[:NO_OF_BYTES]
+            encryption_key_byte = session_key[:NO_OF_BYTES]
```

Comment:
if e_key is not null then use e_key to decrypt the data

Reply:
its handled in else case (line #223) but not visible in the git diff